upstairs/src/lib.rs

```diff
@@ -4268,7 +4729,7 @@ async fn process_new_io(
             let _ = req.send.send(Ok(()));
         }
         BlockOp::QueryUpstairsActive { data } => {
```

Comment:
let's change the block op to QueryGuestIoReady?

Reply:
Good idea, Done in 9608a17.